vision/unit_tests/test_coordinate_lengths.py

```diff
@@ -0,0 +1,151 @@
+"""
+Testing vision.deskew.coordinate_lengths.py
+"""
+
+import unittest
+
+import numpy as np
+
+from vision.deskew.coordinate_lengths import latitude_length
+from vision.deskew.coordinate_lengths import longitude_length
+
+
+class TestCoordinates(unittest.TestCase):
+    """
+    Testing the coordinate lengths module
+    """
+
+    # Testing latitude_length
+
+    # Tests the maximum possible latitude in degrees as an input to the latitude_length function
+    def test_latitude_length_at_max_val(self) -> None:
+        """
+        Asserts that latitude_length returns expected value at maximum input (90 degrees)
+
+        Returns
+        -------
+        None
+        """
+        max_input_in_degrees = 90
```

Comment:
`max_input_in_degrees` needs a type hint. If you need any help with type hints, you can check out our docs on it here: https://missourimrr.github.io/docs/python/type-annotations/full-guide.html